packages/design-tokens/src/js/device/index.ts

```diff
@@ -0,0 +1,166 @@
+// Breakpoint
+export const breakpointMobile = 0;
+export const breakpointTablet = '768px';
+export const breakpointDesktop = '1280px';
+
+// Form field
+export const formFieldSmallRadiusMobile = 'var(--radius-300)';
+export const formFieldSmallRadiusTablet = 'var(--radius-300)';
+export const formFieldSmallRadiusDesktop = 'var(--radius-300)';
+
+export const formFieldMediumRadiusMobile = 'var(--radius-300)';
+export const formFieldMediumRadiusTablet = 'var(--radius-300)';
+export const formFieldMediumRadiusDesktop = 'var(--radius-300)';
+
+export const formFieldLargeRadiusMobile = 'var(--radius-300)';
+export const formFieldLargeRadiusTablet = 'var(--radius-300)';
+export const formFieldLargeRadiusDesktop = 'var(--radius-300)';
+
+// Button
+export const buttonLargeRadiusMobile = 'var(--radius-full)';
+export const buttonLargeRadiusTablet = 'var(--radius-full)';
+export const buttonLargeRadiusDesktop = 'var(--radius-full)';
+
+export const buttonMediumRadiusMobile = 'var(--radius-full)';
+export const buttonMediumRadiusTablet = 'var(--radius-full)';
+export const buttonMediumRadiusDesktop = 'var(--radius-full)';
+
+export const buttonSmallRadiusMobile = 'var(--radius-full)';
+export const buttonSmallRadiusTablet = 'var(--radius-full)';
+export const buttonSmallRadiusDesktop = 'var(--radius-full)';
+
+// Container
+export const containerPaddingMobile = '16px';
+export const containerPaddingTablet = '32px';
+export const containerPaddingDesktop = '32px';
+
+export const containerXsmallMaxWidthMobile = 0;
```

Comment:
please remove all container...maxWidth tokens, they are not responsive and we will remove them from the Figma Device collection as well.